src/command.rs

```diff
@@ -23,12 +23,38 @@ pub enum Command {
         #[structopt(long = "scope", short = "s")]
         scope: Option<String>,
     },
+
     #[structopt(name = "pack")]
     /// 🍱  create a tar of your npm package but don't publish! [NOT IMPLEMENTED]
     Pack { path: Option<String> },
+
     #[structopt(name = "publish")]
     /// 🎆  pack up your npm package and publish! [NOT IMPLEMENTED]
     Publish { path: Option<String> },
+
+    #[structopt(name = "adduser", alias = "login", alias = "add-user")]
```

Comment:
Ah okay you did all them here this is good. I'd swap login and adduser in the arguments

Reply:
sure can do. Shall we keep the function names as adduser?
I feel like changing them to login would make things less confusing.